frontend/lib/Routing/ProfileNavigator.dart

```diff
@@ -1,12 +1,11 @@
+import 'package:adpv_frontend/Repository/EndpointRepository/EndpointGateway.dart';
 import 'package:adpv_frontend/Views/ProfileView.dart';
 import 'package:flutter/material.dart';
 
-import '../Repository/AbstractEndpointRepository.dart';
-
 class ProfileNavigator extends StatefulWidget {
-  final AbstractEndpointRepository endpointRepository;
+  final EndpointGateway endpointGateway;
```

Comment:
Czy jest to nam potrzebne tutaj?

Reply:
Profil pusty, przygotowany do pisania, nie wiem czy potrzebne

Reply:
noo, zostawmy póki co ten profil, tam i tak nic nie ma